jsk_2016_01_baxter_apc/euslisp/main-stow.l

```diff
@@ -39,7 +39,7 @@
           (ros::set-param (format nil "~a_hand/state" (send *ri* :arm-symbol2str arm)) (symbol2str state)))
         (:initialize
           (ros::ros-info "[main] ~a, ~a" arm state)
-          (setq tote-contents (send *ri* :get-tote-contents))
+          (setq tote-contents (get-tote-contents))
```

Comment:
I still think both function can be the method of baxter-interface class, because we can easily find it by `(send *ri* :methods)` and it is capsulized.